docs/source/notebooks/finch-usage.ipynb

```diff
@@ -81,13 +81,15 @@
      "text": [
       "ProcessSucceeded\n",
       "frost_daysResponse(\n",
-      "    output_netcdf='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/out.nc',\n",
-      "    output_log='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/log.txt'\n",
+      "    output_netcdf='http://localhost:5000/outputs/50c0a3f8-67c7-11ea-9e2d-b06ebf31cced/frost-days_SRES-A2-experiment_20460101-20650101.nc',\n",
```

Comment:
Should also keep `https://pavics.ouranos.ca/wpsoutputs` to not break Jenkins later.

Reply:
Ideally those differences would be catched by the `output_sanitize.cfg`, so the tests pass both on Jenkins and Travis-CI.

Reply:
> Ideally those differences would be catched by the `output_sanitize.cfg`, so the tests pass both on Jenkins and Travis-CI.

Agreed, I'll look into this next time I am around notebook tests.

Reply:
This output is actually skipped by a `NBVAL_IGNORE_OUTPUT` comment, so tests will pass either way on jeknins. Which is good, because uuid will never correspond, unless we do some `output_sanitize.cfg` regex.

Reply:
Oh !  Did not realize there is a `NBVAL_IGNORE_OUTPUT`!  Wonder why it was added since we do have regex to sanitize the uuid.

Reply:
Could you paste the regex that skips uuids? It doesn't seem to be in this repo?

Reply:
This repo should curl it from https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/raw/master/notebooks/output-sanitize.cfg so there's only 1 copy of this file.  Raven should also doing this, I hope !  Must have sample code from Raven.

I am actually trying to add regex for this output url here https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/raw/add-regex-to-output-sanitize-cfg-for-Travis-CI/notebooks/output-sanitize.cfg can you help test so we can remove the `NBVAL_IGNORE_OUTPUT`?

Reply:
Oh wait, the `NBVAL_IGNORE_OUTPUT` was for the `print(log)` cell, not the `print(out)` cell that shows the wps output url.

Reply:
What about adding the output-sanitize.cfg to the bird-house cookie cutter? It seems a bit overkill to me to add this curl command for a single cell... and the fact that the cell executed without errors is a good indication that there was no error... I don't know, I'll let you decide if it's worth the effort.

Reply:
We do not refresh from the cookie-cutter fast/frequent enough, so I'd rather do not have the `output-sanitize.cfg` there.

The cell execute without error in Travis-CI, but will not in Jenkins.

What we can do is add the curl command to travis-ci file in the cookie-cutter ! ... if the cookie-cutter has such a file and we are good for all the future notebooks for all the future birds.

Oh wait, the entire notebook passed on Travis-CI even without the regex for the UUID?  What?!!!

Reply:
Looked at Travis-CI logs
```
$ make test-notebooks

make: Nothing to be done for 'test-notebooks'.

The command "make test-notebooks" exited with 0.
```

The notebooks test did not run, so of course it passed !

Reply:
My bad, Raven has the `output-sanitize.cfg` physically committed in the repo https://github.com/Ouranosinc/raven/blob/a4dc8ebb85a7c239596852929509f6aa5b5fa245/docs/source/output_sanitize.cfg and I approved that gigantic PR it was part of.  Brown paper bag on me.

Can you just do it proper this time so there is no duplicate of that file and I'll tell @Zeitsperre to fix on Raven side?

Sorry for not catching earlier on Raven side so you would already have sample code.